app/src/redux/support/intercom-binding.ts

```
@@ -26,7 +26,7 @@ export const bootIntercom = (props: IntercomPayload): void => {
       user_id: userId,
     }
     log.debug('Booting intercom', props)
-    global.Intercom(Constants.INTERCOM_BOOT, iprops)
+    global.Intercom?.(Constants.INTERCOM_BOOT, iprops)
```

Source code change. We've had bugs in the past with intercom silently not being loaded, so this no-op here could hide an actual bug that we'd want to fail loudly

---

👍 I'll update the `global.d.ts` so that `{Intercom?: ... }` isn't defined as optional  -> `{Intercom: ...}`. There is test coverage for Intercom not loading.